src/getCoinCombination.test.js

```diff
@@ -3,8 +3,23 @@
 describe('getCoinCombination', () => {
   const { getCoinCombination } = require('./getCoinCombination');
 
-  it(`should be declared`, () => {
+  it('should be declared', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should return an array', () => {
+    expect(getCoinCombination(0))
+      .toBeInstanceOf(Array);
+  });
+
+  it('should return an array with 4 elements', () => {
+    expect(getCoinCombination(0))
+      .toHaveLength(4);
+  });
+
+  it(`should return [0, 0, 0, 0] for 0`, () => {
```

Comment:
Pls, add more tests. These 3 tests are not enough to cover the task requirements